src/main/docker/go.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+export GATEWAY_OPTS=-Dorg.kaazing.gateway.management.AGRONA_ENABLED=true
```

Comment:
We could use "ENV GATEWAY_OPTS -Dorg.kaazing.gateway.management.AGRONA_ENABLED=true" within the Dockerfile for this purpose